src/ralph/backends/data/clickhouse.py

```diff
@@ -197,6 +193,9 @@ def list(
             self.logger.error(msg, error)
             raise BackendException(msg % error) from error
 
+        if new:
+            self.logger.warning("The `new` argument is ignored")
+
```

Comment:
We should also probably update the docstring for this argument

Reply:
Nice catch) Thanks) Updated the docstring)